examples/surfaceqg_decaying.jl

```diff
@@ -130,12 +135,11 @@ function plot_output(prob)
             yticks = -3:3,
             xlabel = "x",
             ylabel = "y",
-             title = "buoyancy bˢ",
+             title = "buoyancy bₛ",
```

Comment:
I see this as b[box], this notation appears in a few spots. Is it just a rendering issue with the character in Github's document viewer?

Reply:
Yes it's GitHub issue... It's a subscript s. Looks ok in the [docs preview](https://fourierflows.github.io/GeophysicalFlowsDocumentation/previews/PR118/generated/surfaceqg_decaying/).